docs/docs/static-query.md

```diff
@@ -19,13 +19,13 @@ Video hosted on [egghead.io][egghead].
 We'll create a new `Header` component located at `src/components/header.js`:
 
 ```jsx
-import React from "react"
-import { StaticQuery, graphql } from "gatsby"
+import React from 'react';
+import { StaticQuery, graphql } from 'gatsby';
 
-const Header = () => (
+export default () => (
   <StaticQuery
     query={graphql`
-      query {
+      query HeadingQuery {
```

Comment:
A question for @jlengstorf or @amberleyromo - technically we don't need to name queries any more. In this case I guess we should leave this in so the code matches the video?

Reply:
Yeah, I didn't know that when I made the video. We could potentially call it out, but it doesn't do any harm, right? I think we can leave this one for now.